src/js/clickCount.js

```diff
@@ -0,0 +1,34 @@
+import editDescription from './editInLocalStorage.js';
+import List from './list.js';
+import removeTaskFromLocalStorage from './removeTaskFromLocalStorage.js';
+
+const convertTextToInput = (parent, index) => {
+  const input = document.createElement('input');
+  const list = new List();
+  const getData = list.getTaskByIndex(Number(index));
+  if (getData.description) input.value = getData.description;
+  input.className = 'edit-todo';
+  parent.insertBefore(input, parent.firstChild);
```

Comment:
- [ ] Excellent work on the requirements. However, when I try to delete an item, I receive the error shown below.

#### Error :point_down: 
![Screenshot from 2023-05-03 20-02-20](https://user-images.githubusercontent.com/89228743/236036372-430647a6-19ab-467a-b5cf-f9842732247e.png)

The time getData is null or undefined and if the getTaskByIndex method in the List class does not find a task with the specified index that error occurs. :point_down: 
![Screenshot from 2023-05-03 20-02-56](https://user-images.githubusercontent.com/89228743/236036968-24f162a7-b933-4024-8796-a49bee399b66.png)

To fix this error I would suggest you to check if getData is not null or undefined before accessing its description property. 
#### Like this :point_down: 
```
if (getData && getData.description) input.value = getData.description;
```
```suggestion
  const list = new List();
  const getData = list.getTaskByIndex(Number(index));
  if (getData && getData.description) input.value = getData.description;
  input.className = 'edit-todo';
  parent.insertBefore(input, parent.firstChild);
```

Reply:
Hello @ridwanediallo 
Very nice to catch a bug I thought you wouldn't see it :satisfied: but the problem is not in the code you showed 
the Bug founded when I delete the task I update LocalStorage index for tasks and I didn't update the index for item div so the problem is not in the If condition
btw I fixed the problem

Reply:
Yeah, @waelelsafty07, nice try 😄. Actually, if we do not pay close attention, it is possible that we will miss it. Because you can sometimes add and delete everything without getting an error. It took me some time to figure out what was going on and how to deal with it. 
But it's great if you understand how your code works because it will make debugging easier. Excellent work. 💪